os/include/sys/syscall.h

```diff
@@ -291,25 +291,27 @@
 #define SYS_dup                        (__SYS_filedesc + 1)
 #define SYS_dup2                       (__SYS_filedesc + 2)
 #define SYS_fcntl                      (__SYS_filedesc + 3)
-#define SYS_lseek                      (__SYS_filedesc + 4)
-#define SYS_mkfifo                     (__SYS_filedesc + 5)
-#define SYS_mmap                       (__SYS_filedesc + 6)
-#define SYS_open                       (__SYS_filedesc + 7)
-#define SYS_opendir                    (__SYS_filedesc + 8)
-#define SYS_pipe                       (__SYS_filedesc + 9)
-#define SYS_readdir                    (__SYS_filedesc + 10)
-#define SYS_rewinddir                  (__SYS_filedesc + 11)
-#define SYS_seekdir                    (__SYS_filedesc + 12)
-#define SYS_stat                       (__SYS_filedesc + 13)
-#define SYS_statfs                     (__SYS_filedesc + 14)
-#define SYS_telldir                    (__SYS_filedesc + 15)
+#define SYS_fstat                      (__SYS_filedesc + 4)
```

Comment:
System call numbers should be matched with the index of g_funclookup[] and g_stublookup[].
Please, check the sequence of g_funclookup[] and g_stublookup[] as well.

Reply:
In syscall.h and syscall_lookup.h, syscalls are registered in the same order. Please check again.

Reply:
OK. I checked it.